design/open/spark-outputcommitter/lakeFS-output-committer-execution-plan.md

```diff
@@ -0,0 +1,38 @@
+# LakeFS Output Committer - Execution Plan
+[LakeFS Output Committer Proposal](https://github.com/treeverse/lakeFS/blob/master/design/open/spark-outputcommitter/committer.md)
+
+## Overview
+Implementing lakeFS Output Committer meaning implementing the class [FileOutputCommitter](https://hadoop.apache.org/docs/stable/api/org/apache/hadoop/mapreduce/lib/output/FileOutputCommitter.html) and using lakeFS client to create/merge branches as part of setup and commit job/tasks (this is the [sample flow](https://github.com/treeverse/lakeFS/blob/master/design/open/spark-outputcommitter/committer.md#sample-flow))
+
+### Milestone 1
+The first milestone is focused on writing a file with single mode (probably text) and one file format (probably append).
+At this stage we'll implement `commit`, `setup`, `abort` class methods.
+
+### Milestone 2
+This milstone is focused on writing all file formats, in all modes. Also, implement the relevant remaining class methods.
+
+The relevant file formats are:
+* Text
+* CSV
+* Parquet *
```

Comment:
Why do we have a * sign near parquet? 

What do you think of separating parquet + overwrite mode into a separate milestone? the reason why I suggest separating it is that we know that by the end of this milestone we can start talking to users about it and hopefully get feedback.